Catalog/README.md

```diff
@@ -0,0 +1,53 @@
+# Low Code App Editor
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SkylineCommunications_Low-Code-App-Extensions&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SkylineCommunications_Low-Code-App-Extensions)
+
+The **Low Code App Editor** simplifies managing low-code applications within the DataMiner System. It allows you to import, export, edit, and delete apps while offering customization options for metadata, resources, and deployment settings. With features like tailored export configurations and cross-agent synchronization, the tool streamlines app lifecycle tasks, making it a practical solution for maintaining and deploying low-code apps efficiently.
+
+> [!IMPORTANT]
+> After the deploy you'll need to manually replace on of the references.
+>
+> C:\Skyline DataMiner\ProtocolScripts\WebApiLib.dll needs to be C:\Skyline DataMiner\Webpages\API\bin\WebApiLib.dll
+
+![Overview](Images/LCAEditor_1_0_0_13.gif)
+
+## Installation
+
+#### Step 1: Deploy the Low Code App Editor package
+
+1. Click the **Deploy** button to deploy the package directly to your DataMiner System.
```

Comment:
This seems like redundant information, i.e. applicable to any Catalog item.

Reply:
Jeah I copied that part for Edib's example. I actually quite like it being there, because it's only 3 lines and if this is the rare case that it's your first deployment it could be usefull.
[Performance Anayzer](https://catalog.dataminer.services/details/414894ce-21ae-48e7-b2c3-0652fff08349)